modules/gapi/src/backends/plaidml/gplaidmlbackend.cpp

```diff
@@ -198,6 +198,9 @@ void cv::gimpl::GPlaidMLExecutable::run(std::vector<InObj>  &&input_objs,
     exec_->run();
 
     for (auto& it : output_objs) bindOutArg(it.first, it.second);
+
+    // FIXME:
+    // PlaidML backend haven't been updated with RMat support
```

Comment:
Does PlaidML work w/o it?

Reply:
I need to check this

Reply:
It was decided to port plaidML backend to RMat in a separate activity